File: 0867.转置矩阵/solution.cpp
class Solution {
public:
    vector<vector<int>> transpose(vector<vector<int>>& matrix) {
        int row=matrix.size();
        int col=matrix.at(0).size();
        vector<vector<int>> ans(col,vector<int>(row));
        for (int i=0;i<row;i++) {
            for (int j=0;j<col;j++) {
                ans.at(j).at(i)=matrix.at(i).at(j);
            }
        }
        return ans;
    }
};

File: 0867.转置矩阵/solution.go
func transpose(matrix [][]int) [][]int {
	row := len(matrix)
	col := len(matrix[0])
	ans := make([][]int, col)
	for i := range ans {
		ans[i] = make([]int, row)
	}
	for i := 0; i < row; i++ {
		for j := 0; j < col; j++ {
			ans[j][i] = matrix[i][j]
		}
	}
	return ans
}
